lib/content-services/src/lib/content-node-selector/content-node-selector-panel/content-node-selector-panel.component.ts

```diff
@@ -99,6 +99,8 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
         skipCount: 0
     });
 
+    readonly DISABLE_ACTION_FOLDER_LIST = ['-mysites-'];
```

Comment:
no need to have it as uppercase. It can be named as just disableActionFolderList as it's just normal readonly field. Uppercase is mostly used for global constants or for "static readonly" fields but for just readonly field better is using just camel case